### DISCUSSION TITLE
Animated dial

### DISCUSSION THREAD

**Discussion by AlexBinyon99** [Bodmer/TFT_eSPI#2791] [Q&A - Sprites]
i have an issue with ghosting

![IMG20230816110307](https://github.com/Bodmer/TFT_eSPI/assets/113347802/3a9b545c-8e26-41be-9cc1-2e6dcfe5ab6e)
i have chanched it slightly so that it re-uploads the background frequently to combat this. is there a better way to make the sprites "trasparent"

when the code is untouched, it appears like this :
![IMG20230816100739](https://github.com/Bodmer/TFT_eSPI/assets/113347802/353b14cf-ab85-4c76-b9f9-3cd4c24f5d92)

but none of the examples online look like they have the same issue. possibly ive done something wrong

any advice helps

**Comment by Bodmer** [comment]
This suggest the display pixel data cannot be read from the TFT chip. This may be because this is not supported for the interface used or MISO has not been connected for a SPI display.

There is an example that checks if reads are working:
https://github.com/Bodmer/TFT_eSPI/tree/master/examples/Test%20and%20diagnostics/TFT_ReadWrite_Test

If the display canot be read and the processor has sufficient RAM then sprites can be used, see example here:
https://github.com/Bodmer/TFT_eSPI/discussions/860#discussioncomment-208303

**Reply by AlexBinyon99** [reply]
I am using a standard esp32 dev module and wire it 

#define TFT_MOSI 23 // In some display driver board, it might be written as "SDA" and so on.
#define TFT_SCLK 18
#define TFT_CS   16  // Chip select control pin
#define TFT_DC   15  // Data Command control pin
#define TFT_RST  4  // Reset pin (could connect to Arduino RESET pin)
//#define TFT_BL   19  // LED back-light


![image](https://github.com/Bodmer/TFT_eSPI/assets/113347802/608aa63b-e3df-4020-9cff-df6ea837c320)



i have tried the example you posted in the  "Gauge Needles #571 " thread ( [https://github.com/Bodmer/TFT_eSPI/issues/571 ])

this example works well and looks nice. 

my initial intention is to have a second GC9A01 screen driven from the same board. Im guessing if i am struggling with the ammount of onboard ram, then this will be unacheivable. I can always use a second board to drive this gauge seperatly. just need to work out how to communicate between them 
![IMG20230817164915](https://github.com/Bodmer/TFT_eSPI/assets/113347802/2654417b-66a9-44ad-bcf3-11d4221ec0d2)


thank you
